code/modules/mob/inventory.dm

```diff
@@ -292,8 +292,11 @@
 //for when you want the item to end up on the ground
 //will force move the item to the ground and call the turf's Entered
 /mob/proc/dropItemToGround(obj/item/I, force = FALSE)
-	I.pixel_x = rand(-6,6)
-	I.pixel_y = rand(-6,6)
+	if(I)
+		I.pixel_x = rand(-6,6)
+		I.pixel_y = rand(-6,6)
+	else
+		stack_trace("passed a null item to dropItemToGround()")

```

Comment:
Is a null argument actually an error? It seems kind of pointless to require sanity checks every time dropItemToGround() is called when the end result is the same as just bailing on the proc with if(!I) return